src/config.py

```diff
@@ -116,10 +115,8 @@ class Status:
         """
 
         STATUS_READY_FOR_UPGRADE = "status-shows-ready-for-upgrade"
-        CONFIG_SERVER_WAITING_FOR_REFRESH = WaitingStatus("Waiting for refresh command.")
 
     class Upgrade:
         """Upgrade related constants."""
 
-        WAITING_FOR_REFRESH_KEY = "waiting_for_refresh"
```

Comment:
This was an artifact from some old work. It can be removed as it is no longer used